src/test/docbook/pass/typedefsynopsis.001.xml

```diff
@@ -13,14 +13,16 @@
       <db:productnumber>1.2</db:productnumber>
    </db:info>
    <db:classsynopsis>
+     <db:info>
+      <db:releaseinfo role="threadsafety">reentrant</db:releaseinfo>
```

Comment:
Fair enough. I think there's been agreement to add a `meta` element which would be more neutral.

In that example, do you intend the output to be:

```
rentrant
QtCore
#include <qfileinfo.h>
io
shared
```

Because that's what you'll get if they're all `synopsisinfo`.

Reply:
Most of these are not supposed to be rendered as-is, but rather cooked in some way (like a table with the role on the left and the value on the right). They're just metadata about the synopsis, but are not part of it per se (it's just that I don't think there is another place to put this information, and the bound with the synopsis is very strong).